core/store/src/main/java/uk/gov/gchq/gaffer/store/serialiser/lengthvalue/ElementIdSerialiser.java

```diff
@@ -103,4 +103,9 @@ public ElementId deserialiseEmpty() throws SerialisationException {
     public boolean preservesObjectOrdering() {
         return false;
     }
+
+    @Override
+    public boolean isConsistent() {
+        return true;
```

Comment:
This should actually be:
return null != vertexSerialiser && vertexSerialiser.isConsistent();